XRTK.Ultraleap/Packages/com.xrtk.ultraleap/Runtime/Extensions/QuaternionExtensions.cs

```diff
@@ -14,7 +14,14 @@ public static class QuaternionExtensions
         /// Converts a <see cref="Leap.LeapQuaternion"/> object to a <see cref="Quaternion"/> object.
         /// </summary>
         /// <param name="vector">The <see cref="Leap.LeapQuaternion"/> to convert.</param>
-        /// <returns>A <see cref="Quaternion"/>.</returns>
+        /// <returns>A <see cref="Quaternion"/> with identical values to the input <see cref="Leap.LeapQuaternion"/>.</returns>
         public static Quaternion ToQuaternion(this Leap.LeapQuaternion q) => new Quaternion(q.x, q.y, q.z, q.w);
+
+        /// <summary>
+        /// Converts a <see cref="LeapInternal.LEAP_QUATERNION"/> object to a <see cref="Quaternion"/> object.
+        /// </summary>
+        /// <param name="vector">The <see cref="LeapInternal.LEAP_QUATERNION"/> to convert.</param>
+        /// <returns>A <see cref="Quaternion"/> with identical values to the input <see cref="LeapInternal.LEAP_QUATERNION"/>.</returns>
+        public static Quaternion ToQuaternion(this LeapInternal.LEAP_QUATERNION q) => new Quaternion(q.x, q.y, q.z, q.w);
```

Comment:
I think it'd be better to do an implicit cast for these. Check out what I did in the Lumin package. You could update it to implicitly cast from `LeapInternal.LEAP_QUATERNION` to `Quaternion`